snow/engine/snowman/block/vm.go

```diff
@@ -5,12 +5,19 @@ package block
 
 import (
 	"context"
+	"errors"
 
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/snow/consensus/snowman"
 	"github.com/ava-labs/avalanchego/snow/engine/common"
 )
 
+// ErrIndexIncomplete is used to indicate that the VM is currently repairing its
+// index.
+//
+// TODO: Remove after v1.11.x activates.
```

Comment:
just reasoning aloud here: is this true even for the ProposerVM? I mean once we're sure that any VM will implement its height index, there won't be cases there proposerVM height index can be incomplete (following a successful initialization)?

Reply:
The index can still be repaired async until `v1.11.x`